src/Symfony/Component/Console/Question/Question.php

```diff
@@ -38,6 +38,7 @@ class Question
     private ?\Closure $normalizer = null;
     private bool $trimmable = true;
     private bool $multiline = false;
+    private ?int $timeoutSeconds = null;
```

Comment:
```suggestion
    private ?int $timeout = null;
```

same for set and get methods?

Reply:
I see that as making it worse as you need to check implementation to know which unit is used.